.travis.yml

```diff
@@ -1,11 +1,7 @@
 language: rust
 rust:
   - stable
-  - 1.30.1
-  - 1.29.0
-  - 1.28.0
-  - 1.27.2
-  - 1.26.2
+  - 1.32.0
```

Comment:
Stable and 1.32.0 is currently the same thing, so both isn't needed until the next version is released.